index.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Global Female Tech Hackathon</title>
+    <link rel="stylesheet" href="./stylesheets/style.css" type="text/css">
+</head>
+<body>
+    <!-- NAVIGATION SECTION -->
+    <header>
+        <ul id="navTopbar" class="desktopOnly">
+            <li class="navtopTexts"><a href="#"><img src="./media-library/facebook-icon.png" alt="facebook-icon" width="20"></a></li>
+            <li class="navtopTexts"><a href="#"></a><img src="./media-library/twitter-icon.png" alt="twitter-icon" width="20"></a></li>
+            <li class="navtopTexts"><a href="#"></a>English</a></li>
+            <li class="navtopTexts"><a href="#"></a>My Page</a></li>
+            <li class="navtopTexts"><a href="#"></a>Logout</a></li>
+        </ul>
+      <section id="navSection" class="navMainSection">
+        <p id="logo" class="desktopOnly"><a href="#"><img src="./media-library/gft-logo.png" alt="site-logo" width="200" height=""></a></p>
+        <nav id="menuContainer">
+            <p id="menuIcon" class="mobileOnly"><img src="./media-library/gft-menu-icon.png" alt="menu-icon" width="" height=""></p>
+            <ul id="menu" class="menuList">
+                <p id="menuCloseIcon" class="mobileOnly"><img src="./media-library/gft-menu-close-icon.png" alt="menu-close-icon" width="" height=""></p>
+                <li class="menuItems"><a href="./about.html">About </a></li>
+                <li class="menuItems"><a href="#">Program</a></li>
+                <li class="menuItems"><a href="#">Sponsor</a></li>
+                <li class="menuItems"><a href="#">News</a></li>
+                <li class="menuItems"><a href="#">Contact</a></li>
+                <li id="menuBtn" class="outlineBtns animBtns">Register</li>
```

Comment:
- [ ] **Congratulations 👏 👏, You have made most of the suggested changes 👍** However, as suggested by the previous reviewer, It would be better to avoid text overwraps on the footer by clicking the hamburger button 🙏, You may Kindly set the `z-index` to make sure that the hamburger menu hides all the page's contents for the best user experience 👍 


![image](https://github.com/geekelo/gft-hackathon-webapp-project/assets/102757126/21296f7b-8f54-4462-900f-69231c8eef4e)